src/DotNet.Status.Web.Tests/GitHubDeserializationTests.cs

```diff
@@ -37,31 +39,76 @@ private static string GetTestPayload()
             return reader.ReadToEnd();
         }
 
+        public User CreateUser(string login)
+        {
+            return new User(
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                login,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default,
+                default);
+        }
+
         [Test]
         public void IssueEventDeserialization_SameAsExpected()
         {
             var expected = new IssuesHookData
             {
                 Action = "labeled",
-                Issue = new IssuesHookIssue
-                {
-                    Assignee = new IssuesHookUser
+                Issue = new Octokit.Issue(
```

Comment:
The point of the model was to abstract away all this noise (also also make it so we didn't have to serialize giant objects into memory that we aren't going to look at).

Reply:
Thats what octokit already does though, if the large number of parameters is the problem, I can make some factory methods. I don't like that we basically rewrote the model classes for the notifications when they already exist.

Reply:
True true, I'm fine. i think the might not have existed way back in like octokit 20 or whatever when I wrote this (I tried them, and somthing I can't recall didn't work)

Reply:
Maybe they didn't serialize right? They are built with no attributes to say what the proper json key is. The serializer built into octokit knows how to handle the naming.

Reply:
Oooo. That might have been it. :-)